docs/getting-started.md

```diff
@@ -25,24 +25,6 @@ docker run -it -u $UID:$GID -v $PWD:/path checkmarx/kics:ubi8 scan -p /path/asse
 
 Another option is [rebuilding the dockerfile](https://github.com/Checkmarx/kics/blob/master/Dockerfile.ubi8) providing build arguments e.g: `--build-arg UID=999 --build-arg GID=999 --build-arg KUSER=myuser --build-arg KUSER=mygroup`
 
-#### Homebrew
-
-KICS is avaiable on Checkmarx [homebrew-tap](https://github.com/Checkmarx/homebrew-tap). It can be used as follows:
-
-```
-brew install Checkmarx/tap/kics
-```
-
-To use KICS default queries add KICS_QUERIES_PATH env to your `~/.zshrc`, `~/.zprofile`:
-
-```
-echo 'export KICS_QUERIES_PATH=/usr/local/opt/kics/share/kics/assets/queries' >> ~/.zshrc
-```
```

Comment:
Hello, @konstruktoid!

Thank you so much for noticing, reporting, and improving the documentation! 

We would like to ask you if you could main/update the Homebrew note because some users are still using versions older than 1.5.2. As a suggestion:


#### [Deprecated] Homebrew

KICS is available on Checkmarx [homebrew-tap](https://github.com/Checkmarx/homebrew-tap) **only for versions until 1.5.1**. It can be used as follows:

```
brew install Checkmarx/tap/kics
```

To use KICS default queries add KICS_QUERIES_PATH env to your `~/.zshrc`, `~/.zprofile`:

```
echo 'export KICS_QUERIES_PATH=/usr/local/opt/kics/share/kics/assets/queries' >> ~/.zshrc
```